src/entities/sprint.ts

```diff
@@ -0,0 +1,34 @@
+import { ReducedAgile, ReducedAgileImpl } from "./agile";
+import { ReducedIssue, ReducedIssueImpl, Issue, IssueImpl } from "./issue"
+
+export class ReducedSprintImpl {
+    id?: string = '';
+    name?: string = '';
+    goal?: string = '';
+    start?: number = 0;
+    finish?: number = 0;
+    archived: boolean = false;
+    agile?: ReducedAgile = new ReducedAgileImpl();
+    issues: ReducedIssue[] = [new ReducedIssueImpl()];
+    unresolvedIssuesCount: number = 0;
+    previousSprint?: ReducedSprint = undefined;
+}
+
+export class SprintImpl extends ReducedSprintImpl {
+    issues: Issue[] = [new IssueImpl()];
```

Comment:
I'd use the ReducedIssue here since full Issues are quite big, youtrack may not respond very fast. In the ReducedSprintImpl I'd not include any issues.